tensorrt_llm/_torch/visual_gen/models/cosmos3/guardrails.py

```
@@ -0,0 +1,371 @@
+
+        classifier = SafetyClassifier(input_size=1152, num_classes=7)
+        ckpt_path = os.path.join(
+            guardrail_ckpt_dir, "video_content_safety_filter", "safety_filter.pt"
+        )
+        checkpoint = torch.load(ckpt_path, map_location="cpu", weights_only=True)
+        state = {k.removeprefix("network."): v for k, v in checkpoint["model"].items()}
+        classifier.load_state_dict(state)
+        classifier = classifier.to("cuda", dtype=torch.float32).eval()
+
+        def _safety_check(frames: np.ndarray) -> tuple[bool, str]:
+            nonlocal siglip_model, classifier
```

Flake8 (7.3.0)

[error] `nonlocal siglip_model` is unused: name is never assigned in scope

(F824)